cmd/cli/kubectl-kyverno/processor/policy_processor.go

```diff
@@ -194,7 +194,7 @@ func (p *PolicyProcessor) ApplyPoliciesOnResource() ([]engineapi.EngineResponse,
 				}
 				responses = append(responses, generateResponse)
 			}
-			p.Rc.addGenerateResponse(p.AuditWarn, resPath, generateResponse)
+			p.Rc.addGenerateResponse(p.AuditWarn, generateResponse)
```

Comment:
do we need to keep changes in this file ?

Reply:
Please clarify if I didn't get you here. I removed the `resPath` argument because it was being passed but left unused in the `addGenerateResponse` method.